src/rules/__tests__/lowercase-name.test.ts

```diff
@@ -1,9 +1,11 @@
-'use strict';
+import { RuleTester as ESLintRuleTester } from 'eslint';
+import { TSESLint } from '@typescript-eslint/experimental-utils';
+import rule from '../lowercase-name';
 
-const { RuleTester } = require('eslint');
-const rule = require('../lowercase-name');
+const RuleTester: TSESLint.RuleTester = ESLintRuleTester as any;
 
 const ruleTester = new RuleTester({
+  parser: '@typescript-eslint/parser',
```

Comment:
I find it odd they require this - even if the rule is written in TS, it might not necessarily only run with that parser. Might be needed for all I know 🤷‍♂